RunWorkFlow_SWASH.py

```diff
@@ -83,10 +83,11 @@ def Master_SWASH_run(inputDict):
 
     # ________________________________________________ RUN LOOP ________________________________________________
     for timeSegment in dateStringList:
-        fileHandling.makeCMTBfileStructure(path_prefix=path_prefix, date_str=timeSegment)
+#        fileHandling.makeCMTBfileStructure(path_prefix=path_prefix, date_str=timeSegment)
```

Comment:
why'd you pull this out?

Reply:
I just moved it in the file.  Should be a few lines down.  Probably need to pull the fix for ":" in time into the directory listing.